### ISSUE TITLE
Quick Copy in PDF reader should use parent item

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2023]
https://forums.zotero.org/discussion/88746/beta-pdf-reader-copy-reference-and-navigation

Any reason not to do this?

**Comment by mrtcode** [comment]
I guess we want to add not only "Copy Citation" but "Copy Bibliography" as well?

Also, I think it would make sense if copied citation would be pastable into note-editor as an active citation not just plain text.

**Comment by dstillman** [comment]
Yes, it should be equivalent to when the parent item is selected in the library view — both shortcuts (A and C) should work, and both menu options should be available.
